node/resolvers/search/productSearch.ts

```diff
@@ -23,7 +23,20 @@ export const resolvers = {
       const quantity = resources.split('/')[1]
       return parseInt(quantity, 10)
     },
-    products: path(['productsRaw', 'data']),
+    products: ({
+      translatedArgs: { skipSimulation },
+      productsRaw: { data },
+    }: ProductSearchParent) => {
+      return data.map(product => {
+        return {
+          ...product,
+          items: product.items.map(item => ({
+            ...item,
+            skippedSimulation: !!skipSimulation,
```

Comment:
Skip simulation is a boolean, `!!` isn't needed

Reply:
it might be null or undefined, lets say someone pass the wrong value in the resolver above the tree. I think this is safer.